macros/base/start_print.cfg

```diff
@@ -25,7 +27,11 @@ gcode:
     {% set INITIAL_TOOL = params.INITIAL_TOOL|default(0)|int %} # Initial tool (for the MMU/ERCF initialization)
     {% set MATERIAL = params.MATERIAL|default(printer["gcode_macro _USER_VARIABLES"].print_default_material)|string %} # Material type set in the slicer
     {% set FL_SIZE = params.SIZE|default("0_0_0_0")|string %} # Get bounding box of the first layer for the adaptive bed mesh
-    {% set CHECK_GATES = params.CHECK_GATES|default(printer['gcode_macro _USER_VARIABLES'].mmu_check_gates_on_start_print)|int %} # Check if MMU gates are availables
+    {% if klippain_mmu_enabled %}
+        {% set CHECK_GATES = params.CHECK_GATES|default(printer['gcode_macro _USER_VARIABLES'].mmu_check_gates_on_start_print)|int %} # Check if MMU gates are availables
+    {% else %}
+        {% set CHECK_GATES = 0 %}
```

Comment:
Not sure if this is necessary :)
and it's no use...

Reply:
It is, because of the following

`SET_GCODE_VARIABLE MACRO=START_PRINT VARIABLE=check_gates VALUE={CHECK_GATES}`

Reply:
i only added it because if i don't set CHECK_GATES the SET_GCODE_VARIABLE will fail.
I intially had it in until the error came up. so it is a fix of a followup error.